src/layouts/azerty.rs

```diff
@@ -324,7 +324,9 @@ impl KeyboardLayout for Azerty {
                 }
             }
             KeyCode::SemiColon => {
-                if modifiers.is_shifted() {
+                if map_to_unicode && modifiers.is_ctrl() {
+                    DecodedKey::Unicode('\u{000D}')
```

Comment:
The layout would then have two keys producing `\u000D` with `ctrl` because of `KeyCode::M`. I think your change is correct and `KeyCode::M` should not produce `\u000D` on azerty. What do you think?